docs/release-notes/rl-0.8.md

```diff
@@ -346,6 +346,7 @@
 - Fix default telescope ignore list entry for '.git/' to properly match
 - Add [gitlinker.nvim] plugin to `vim.git.gitlinker-nvim`
 - Add [nvim-treesitter-textobjects] plugin to `vim.treesitter.textobjects`
+- Disable Conform for Rust if rust-analyzer is enabled
```

Comment:
Can we have a note on that this is the default behaviour, and that it can be overriden if the user desires. A nested point detailing why and instructing how to "fix" this behaviour would be appreciated.

Reply:
Done.